### ISSUE TITLE
Can't scroll to bottom on Android 

### ISSUE THREAD

**Issue by Sebastian1989101** [daniel-luberda/DLToolkit.Forms.Controls#105]
On Android the scroll stops at about 70% on the way down. It's not possible to scroll through all elements. I use Grid's in my list, each about 300x180 pixel but some are bigger or smaller. All grid sizes are calculated with auto rows and columns (the width is always 300). On Android the scroll bar at the right goes to the bottom while the elements in the list are only scrolled for about 70% at this point. On iOS this works fine (sometimes it needs a few scroll attemps to work there).

**Comment by Sebastian1989101** [comment]
Still having the issue and get many reports from my users that this especially is not working on Android Tablets..

**Comment by daniel-luberda** [comment]
Hi, it probably some kind of `Xamarin.Forms.ListView` issue, which `FlowListView` is based on. Does it work on a standard ListView?

**Comment by Sebastian1989101** [comment]
It is working with the default ListView just fine. This is the code of a tabbed page which contains 4 lists on different pages. Each page has this issue there. I'm currently trying to rebuilt it with Padding instead of Margin because the Margin is somehow not calculated in the RowHeight in Xamarin. 

https://pastebin.com/Nk97e3Dv

**Comment by daniel-luberda** [comment]
I'll try to debug it. Meanwhile, you could also try the newest nuget version and see if it would help.

**Comment by Sebastian1989101** [comment]
Just done a few quick tests. On the first 4 devices it seems to work. I will prepare a Beta update for my app and let my beta users test it a bit more.

**Comment by Sebastian1989101** [comment]
Is this completely fixed in some release now or just closed because it works partially?

**Comment by daniel-luberda** [comment]
I just assumed that it's fixed if you can't reproduce it

**Comment by Sebastian1989101** [comment]
As mention in the edit of my last post it does not come up the same way as before but it still has a problem if I rotate the device while the list is open. So at least it seems to work as long as I do not rotate the device with the latest version.

**Comment by daniel-luberda** [comment]
As a temporary fix you could try this (notice using ItemsSource instead of FlowItemsSource): 

```C#
                var items = (IEnumerable<object>)flowListView.ItemsSource;
                flowListView.ScrollTo(items.ToList().Last(), ScrollToPosition.End, false);
```

If it won't work, it's probably a Xamarin.Forms.ListView issue (on which FlowListView is based)

**Comment by daniel-luberda** [comment]
Did you try it?

**Comment by Sebastian1989101** [comment]
I did not tested it yet. But it seems like the changes to my code are enough to make it work. It looks like the FlowListView has the same problem with margins as many other controls as well (because it is not calculated into the actual control size). Since I rebuilt the list with paddings it looks like the list works even on a size switch (landscape / portrait). Strange thing that it works on iOS the whole time. I just released a new version to my Android beta testes with the padding instead of margin.. lets see if the bug is fixed on their devices too.
